x-pack/legacy/plugins/siem/public/components/ml/tables/anomalies_host_table.tsx

```diff
@@ -51,9 +50,9 @@ export const AnomaliesHostTable = React.memo<AnomaliesHostTableProps>(
       narrowDateRange
     );
     const pagination = {
-      pageIndex: 0,
-      pageSize: 10,
-      totalItemCount: getSizeFromAnomalies(tableData),
+      initialPageIndex: 0,
+      initialPageSize: 10,
+      totalItemCount: hosts.length,
```

Comment:
Thanks for the extra fixes, I did accidentally use the incorrect props.

I confirmed these you changed them to are correct from EUI code base:
https://github.com/elastic/eui/blob/master/src/components/basic_table/in_memory_table.js#L67

And their docs:

<img width="1063" alt="Screen Shot 2019-10-18 at 8 32 37 AM" src="https://user-images.githubusercontent.com/1151048/67103413-4d833e00-f182-11e9-8d46-7f44cf2979d9.png">
 

You have a great 👁!